api/tests/Feature/NotificationTest.php

```diff
@@ -281,4 +299,168 @@ public function testDateRangeFilter()
+    {
+        if (! config('feature.status_notifications')) {
+            $this->markTestSkipped('This test uses features behind the FEATURE_STATUS_NOTIFICATIONS flag.');
+        }
+
+        $screenInTime = config('constants.far_past_datetime');
+
+        // set up submitted candidate
+        $this->poolCandidate->pool_candidate_status = PoolCandidateStatus::NEW_APPLICATION->name;
+        $this->poolCandidate->saveQuietly(); // don't generate events during set up
+
+        Carbon::setTestNow($screenInTime);
+
+        // simulate screening in
+        $this->poolCandidate->pool_candidate_status = PoolCandidateStatus::SCREENED_IN->name;
+        $this->poolCandidate->save();
+
+        // gather expected notification ID
+        $notificationId = $this->candidateUser->notifications()->sole()->id;
+
+        // check for a notification
+        $this->actingAs($this->candidateUser, 'api')
+            ->graphQL(
+                /** @lang GraphQL */
+                '
+                query myNotifications {
+                        notifications {
+                            data {
+                                id
+                                readAt
+                                createdAt
+                                updatedAt
+                                ... on PoolCandidateStatusChangedNotification {
+                                    oldStatus
+                                    newStatus
+                                    poolId
+                                    poolName {
+                                        en
+                                        fr
+                                    }
+                                }
+                        }
+                    }
+                }
+            '
+            )
+            ->assertJson([
+                'data' => [
+                    'notifications' => [
+                        'data' => [
+                            [
+                                'id' => $notificationId,
+                                'readAt' => null,
+                                'createdAt' => $screenInTime,
+                                'updatedAt' => $screenInTime,
+                                'oldStatus' => PoolCandidateStatus::NEW_APPLICATION->name,
+                                'newStatus' => $this->poolCandidate->pool_candidate_status,
+                                'poolId' => $this->poolCandidate->pool->id,
+                                'poolName' => [
+                                    'en' => $this->poolCandidate->pool->name['en'],
+                                    'fr' => $this->poolCandidate->pool->name['fr'],
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+            ]);
+    }
+
+    /**
+     * Test that use can dismiss notifications
+     *
+     * @return void
+     */
+    public function testCanDismissNotifications()
```

Comment:
Okay, sorry, I just took a closer look at this test, and... I think we can delete it? It seems like just a more complicated version of `testReadNotificationMutation`. Please let me know if I'm missing something.

Reply:
Looks like it was just a different way of doing the same thing so I removed it as part of [reorganized notification tests](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9694/commits/179699d422207e642706fd900a0538072d03a175)